src/pdn/test/regression_tests.tcl

```diff
@@ -77,3 +77,18 @@ record_tests {
 
   existing
 }
+
+set extra_tests {
```

Comment:
What does this do? most of those files are no longer present, so the only ones that should be there is power_switch* and those should be active in the actual regression tests (even if they are using the legacy calls)

Reply:
Yes, these tests were removed, are you saying that the replacement tests fully cover everything that was being checked before? Or are you saying that these aren't really unit tests, but are more like integration tests?

Clearly the power_switches tests need to exist, but with the presence of the C++ code, isnt this code now orphaned? Should I expect to be able to run pdngen with only the TCL code implementation and bypass the C++ code (on a temporary basis). I dont see how I can do that when the TCL commands only seed the C++ code, not the TCL code.

If the test isnt going to run cleanly, then the CI checks will fail. I dont want to checkin the results of the failure just to get a clean CI run.

Reply:
@Colin-Holehouse the tests that are there only handle the c++ code, since the plan is to deprecate the TCL code eventually (but keeping it for a while to allow for some transition). I think you could add the power_switch tests that use the TCL code into the regression (those should not fail and will only generate a warning, which is okay to ignore), that way they are part of the tests and once the C++ code is added to handle the power_switches then those tests can be converted to use the c++ code.  The CFG file can still be provided to setup the TCL pdngen. ie. `pdngen power_switch.cfg` will trigger the TCL code version to run.

Reply:
Ok - thanks, I can get the power_switch tests to run using the base TCL code without going through the C++ code base.